hca/staging_area_validator.py

```diff
@@ -360,10 +369,12 @@ def validate_uuid(self, value: str) -> None:
 
 class SchemaValidator:
     @classmethod
-    def validate_json(cls, file_json: JSON, schema: Optional[JSON] = None) -> None:
+    def validate_json(
+        cls, total_retries, file_json: JSON, schema: Optional[JSON] = None
+    ) -> None:
```

Comment:
```suggestion
    def validate_json(cls,
                       file_json: JSON,
                       total_retries: int,
                       schema: Optional[JSON] = None
                       ) -> None:
```
The `file_json` parameter makes more sense as the first parameter to `validate_json`